### PR TITLE
added more project details

### PR DESCRIPTION
**Pull request by kenyajima**
@dsorkin thank you for reviewing my changes.

### CODE REVIEW

**Comment by dsorkin** [comment]
:+1: Looks pretty good. :shipit:

File: README.md [hunk 1]
```diff
@@ -1 +1,12 @@
...
+
+## Section 1
+
+Brief summary of the project goals.
+
+## Section 2
+
+* list item 1
+* list item 2
+
```
Comment (dsorkin) [inline README.md:11]: Might want a third list item here

File: README.md [hunk 2]
```diff
@@ -1 +1,12 @@
 # hello-world
+
+## Section 1
+
+Brief summary of the project goals.
+
+## Section 2
```
Comment (kenyajima) [inline README.md:7]: Section 2 should start with schedule description.

File: README.md (hunk 1 above)
Comment (kenyajima) [inline README.md:11]: will do.

**Pull request merged**